botathon/register.php

```diff
@@ -98,15 +98,16 @@
             <div class="range">
               <div class="cell-lg-6">
                 <h1>Register</h1>
-                <h6>You may sign up for our <strong>Botathon Season 2</strong> competition below.</h6>
+				<h6>Registration is over for the year. Information about season 3 will be relased during the spring semester 2022. </h6>
+                <!--<h6>You may sign up for our <strong>Botathon Season 2</strong> competition below.</h6>-->
               </div>
 				
-				<div>
+				<!--<div>

```

Comment:
Could you make this one single comment, rather than several different comments? It just makes it harder for the developer who restores the register page if they have to go and find multiple sections to uncomment :)